src/OrbitGl/Track.cpp

```diff
@@ -121,19 +60,16 @@ void Track::DoDraw(Batcher& batcher, TextRenderer& text_renderer, const DrawCont
   float label_width = layout_->GetTrackTabWidth();
   float half_label_width = 0.5f * label_width;
   float tab_x0 = x0 + layout_->GetTrackTabOffset();
-
   const float indentation_x0 = tab_x0 + (indentation_level_ * layout_->GetTrackIndentOffset());
-  Box box(Vec2(indentation_x0, y0), Vec2(label_width, label_height), track_z);
-  batcher.AddBox(box, track_background_color, shared_from_this());
 
   Vec2 track_size = GetSize();
 
   // Draw rounded corners.
-  if (!picking && draw_background_) {
+  if (!picking && draw_background) {
     float radius = std::min(layout_->GetRoundingRadius(), half_label_height);
     radius = std::min(radius, half_label_width);
     uint32_t sides = static_cast<uint32_t>(layout_->GetRoundingNumSides() + 0.5f);
-    auto rounded_corner = GetRoundedCornerMask(radius, sides);
+    auto rounded_corner = orbit_gl::GetRoundedCornerMask(radius, sides);
```

Comment:
As above, we should really be in the orbit_gl namespace already.

Reply:
Yeah, I didn't want to introduce this in the same PR though.